resources/dns/records.tf

```diff
@@ -0,0 +1,379 @@
+  name     = "subspace.network"
+  comment  = "MX record pointing to our preferred mailserver"
+  priority = 5
+  proxied  = false
+  ttl      = 3600
+  type     = "MX"
+  value    = "alt2.aspmx.l.google.com"
+  zone_id  = data.cloudflare_zone.cloudflare_zone.id
+}
+
+resource "cloudflare_record" "mailserver_3" {
+  name     = "subspace.network"
+  comment  = "MX record pointing to our preferred mailserver"
+  priority = 10
+  proxied  = false
+  ttl      = 3600
+  type     = "MX"
+  value    = "alt4.aspmx.l.google.com"
+  zone_id  = data.cloudflare_zone.cloudflare_zone.id
+}
+
+resource "cloudflare_record" "mailserver_4" {
+  name     = "subspace.network"
+  comment  = "MX record pointing to our preferred mailserver"
+  priority = 10
+  proxied  = false
+  ttl      = 3600
+  type     = "MX"
+  value    = "alt3.aspmx.l.google.com"
+  zone_id  = data.cloudflare_zone.cloudflare_zone.id
+}
+
+resource "cloudflare_record" "mailserver_default_domain_key" {
+  name    = "default._domainkey"
+  comment = "DomainKeys authentication for mail"
+  proxied = false
+  ttl     = 3600
+  type    = "TXT"
+  value   = "v=DKIM1; k=rsa; p=MIIBIjANBgkqhkiG9w0BAQEFAAOCAQ8AMIIBCgKCAQEA7zwrsOLB+eJ9SG1t7+OwOT2BuacTImFHozl8I/ mypg7rhtp4i1NpkSnjjDC3FdXXiUhsHTvAUvg5nMtGp3nCwwQYna0C8Jo7dbt3+NUVLj9KCBBBegxPS/WoJghPSbiKq4T/SBdM0K ShrVn7C/1blWA+N4XxOwmVtELV8POMwRYCzlrxCi3kdjbRY+4gXYKmcc7MSRi5ubyR7P/+K1/CkLbJa1SbxdS6/zMRIzPH/6vOR6 be1Qkw5PFsYu0gYbz3QDqYxaUTS3euWSPE3uLnPQEgAryX3SlKQB8uyjNbxF86ukslwwe9Q5cCZ1UPsl/89qcBRirnJbfoGlwb5j rX3QIDAQAB;"
+  zone_id = data.cloudflare_zone.cloudflare_zone.id
+}
+
+resource "cloudflare_record" "mailserver_dmarc" {
+  name    = "_dmarc"
+  comment = "DMARC for communications"
+  proxied = false
+  ttl     = 3600
+  type    = "TXT"
+  value   = "v=DMARC1; p=reject; pct=100; rua=mailto:g5oqy2di@ag.dmarcian.com, mailto:admin@subspace.network; ruf=mailto:admin@subspace.network"
+  zone_id = data.cloudflare_zone.cloudflare_zone.id
+}
+
+resource "cloudflare_record" "mailserver_domain_key" {
+  name    = "_domainkey"
+  comment = "DomainKeys authentication for mail"
+  proxied = false
+  ttl     = 3600
+  type    = "TXT"
+  value   = "v=DKIM1; o=~"
+  zone_id = data.cloudflare_zone.cloudflare_zone.id
+}
+
+resource "cloudflare_record" "mailserver_google_domain_key" {
+  name    = "google._domainkey"
+  comment = "DomainKeys authentication for mail specific to Google mailserver"
+  proxied = false
+  ttl     = 3600
+  type    = "TXT"
+  value   = "v=DKIM1; k=rsa; p=MIIBIjANBgkqhkiG9w0BAQEFAAOCAQ8AMIIBCgKCAQEAyqbz3f5O6pOl9LK3zOmKkczcK9aCdB34tSzDkA ZRD0g+OtBhlyWyzNeQGsrVianIfM9xyfZ7MVJK7sB/VGKIFIe5glb/Lh9tf/kLCRbkOnaafiXP5tOk4DC+mBpHOjyT0GgX5x4hxg oLmeJOrRLSu1niPQY/VqMtNsYa9+gAIo0YZnZeNi2w3FjWaslm5F7uI6mISdH3HZchqhzx2E6Ct+VJhLM0Ir+6v3qV5ylArtJzc+ PoTGdk65n5oq+Ioj9oTK4VmJunZ6jaU2Vimo+2TTAhBtRWQCa8Xy15/3kVZXiKJ4ddtPDnzSEval4er4SksIhtVYaxkeanU7pGbK w1SQIDAQAB"
+  zone_id = data.cloudflare_zone.cloudflare_zone.id
+}
```

Comment:
This is the same as above (except `;` is added unnecessarily at the end of the above record). I don't think this should be necessary, seems accidental, @ImmaZoni do you know who set this up?